var/spack/repos/builtin/packages/cuda/package.py

```diff
@@ -569,3 +569,8 @@ def libs(self):
             if "compat" not in parts and "stubs" not in parts:
                 filtered_libs.append(lib)
         return LibraryList(filtered_libs)
+
+    @property
+    def non_bindable_shared_objects(self):
+        # Avoid binding stub libraries by absolute path
+        return ["stubs"]
```

Comment:
Not that it *really* matters but why not start off with an actual property?  Do you expect many of these to need a whole method?

```suggestion
    #: Avoid binding stub libraries by absolute path
    non_bindable_shared_objects = ["stubs"]
```